### ISSUE TITLE
Class 'GuzzleHttp\Client' not found

### ISSUE THREAD

**Issue by okakzai** [glooby/pexels#3]
Fatal error: Class 'GuzzleHttp\Client' not found in C:\xampp56\htdocs\pexels\src\Client.php on line 32

**Comment by NickCoolii** [comment]
@okakzai do you use composer?
I forked this repo because I had to update Guzzle version to that, which I use at my Laravel project.
But now I have an issue with Authorization:
`Client error: "GET http://api.pexels.com/v1/search?query=london&per_page=15&page=1"
 resulted in a "401 Unauthorized" response: { "error": "Authorization field missing" }`

Updated my fork for work with Guzzle 6.3.3.

Add to composer.json:
```
    "repositories": [
      {
        "type": "vcs",
        "url": "https://github.com/NickCoolii/pexels"
      }
    ],
```
Change at "require" section for `"iceqi/pexels"` to
`"iceqi/pexels": "dev-master"`

Than just make `composer update iceqi/pexels`

**Comment by jcandan** [comment]
@NickCoolii, you may find #5 helpful for your authorization error.

**Comment by NickCoolii** [comment]
@jcandan thanks, but I don't have any error, because using my own fork.
I'm not sure in that this repo alive.

**Comment by kilhage** [comment]
@jcandan @NickCoolii I merged #5 and made a new release. Hope it solves your problems
